app/src/main/java/com/example/tawazz/room/JoinRoom.java

```diff
@@ -0,0 +1,36 @@
+package com.example.tawazz.room;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.NavController;
+import androidx.navigation.Navigation;
+
+import com.example.tawazz.R;
+import com.example.tawazz.utils.gui.NavigationOnClickListener;
+
+public class JoinRoom extends Fragment {
```

Comment:
very confusing that the name of the class doesnt end with `fragment`, like in MenuFragment.